migrator/migrator.go

```diff
@@ -73,6 +73,10 @@ func (m Migrator) FullDataTypeOf(field *schema.Field) (expr clause.Expr) {
 		expr.SQL += " NOT NULL"
 	}
 
+	if field.AutoIncrement {
+		expr.SQL += " AUTO_INCREMENT"
+	}
+
```

Comment:
not suitable for postgres

Reply:
I have not use that db, can you help me to implement that feature?